src/services/notificationService.ts

```diff
@@ -163,6 +163,19 @@ const activityCreator = (payload: any, orttoEventName: NOTIFICATIONS_EVENT_NAMES
         "str:cm:userid": payload.userId?.toString(),
       }
       break
+    case NOTIFICATIONS_EVENT_NAMES.NOTIFY_REWARD_AMOUNT:
+      attributes = {
+        "int:cm:round": payload.round,
+        "str:cm:date": payload.date,
+        "str:cm:amount": payload.amount,
+        "str:cm:contractaddress": payload.contractAddress,
+        "str:cm:farm": payload.farm,
+        "str:cm:message": payload.message,
+        "str:cm:network": payload.network,
+        "str:cm:script": payload.script,
+        "str:cm:transactionhash": payload.transactionHash,
+      }
+      break
     default:
       logger.debug('activityCreator() invalid event name', orttoEventName)
       return;
```

Comment:
@mohammadranjbarz 
I think we need to add notification type for the giv-token-contracts service to the database. (based on line 223 of this file that we call getNotificationTypeByEventNameAndMicroservice() function)

Reply:
@ae2079  Yes please do it ( write a migration for that)